README.md

```diff
@@ -192,6 +192,8 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 ### Rust
 - [Rusty Web](https://davidmcneil.github.io/the-rusty-web/)
 - [parity-wasm - WebAssembly interpreter, decoder and encoder in pure Rust](https://github.com/paritytech/parity-wasm)
+- [rustwasm book](https://rustwasm.github.io/docs/book/)
```

Comment:
Maybe we should put this link in the _Books_ section. What do you think?

Reply:
@mbasso great idea!